recipe-server/client/control_new/components/recipes/RecipeListing.js

```diff
@@ -156,9 +158,27 @@ export default class RecipeListing extends React.Component {
     push(getCurrentURL({ page }));
   }
 
-  handleRowClick(record) {
-    const { push } = this.props;
-    push(`/recipe/${record.id}`);
+  handleRowClick(record, index, event) {
+    const linkTarget = event.target.tagName === 'A';
```

Comment:
`linkTarget` is the name of a thing, not a boolean. Can we call this something more apt to its usage? Or remove the name altogether and replace the check below with the below?

```js
if (event.target.tagName === 'A') {
  return;
}`